src/Markdown/Heading.elm

```diff
@@ -2,7 +2,7 @@ module Markdown.Heading exposing (..)
 
 import Markdown.RawBlock exposing (Attribute, RawBlock(..), UnparsedInlines(..))
 import Parser
-import Parser.Advanced as Advanced exposing ((|.), (|=), Nestable(..), Step(..), andThen, chompWhile, getChompedString, succeed, symbol)
+import Parser.Advanced as Advanced exposing ((|.), (|=), Nestable(..), Step(..), andThen, chompIf, chompWhile, getChompedString, spaces, succeed, symbol)
```

Comment:
I imported chompIf while trying to achieve this PR but is currently unused. Should I remove it for now?

Reply:
Good question. I think it's reasonable to leave it in to avoid going back and forth in the import line. I like to avoid changing my import lines frequently, so that's the rule of thumb I go by. So this looks good 👍